repo.go

```diff
@@ -16,6 +16,7 @@ import (
 
 type repo struct {
 	*url.URL
+	subdir string
```

Comment:
This doesn't seem to open code-server to the subdirectory, are we planning to handle that in a separate PR?

Reply:
Yeah see https://github.com/cdr/sail/issues/72#issuecomment-495434884. Right now it just always defaults to the base directory of the project